airflow/www/extensions/init_appbuilder.py

```diff
@@ -560,7 +561,7 @@ def security_converge(self, dry=False) -> Dict:
         :param dry: If True will not change DB
         :return: Dict with all computed necessary operations
         """
-        return self.sm.security_converge(self.baseviews, self.menu, dry)
+        return self.sm.security_converge(self.baseviews, self.menu, dry)  # type: ignore
```

Comment:
What is the Mypy error this is ignoring?

Reply:
I think, it was complaining about security_covererage is not part of sm. As sm is dynamically loaded I set that to ignore

Reply:
It may be better to fix the `sm = None` definition above (to `sm: AirflowSecurityManager`).

Reply:
Fixed